docker/test-images/zipkin-eureka/pom.xml

```diff
@@ -50,15 +50,6 @@
       <artifactId>spring-cloud-starter-netflix-eureka-server</artifactId>
     </dependency>
 
-    <dependency>
-      <groupId>org.springframework.security</groupId>
```

Comment:
note this didn't change the image size as even if the eureka layer is 55mb the current classpath requires modules not in our JRE, so we use the JDK image as noted in the Dockerfile. The image sum with the JDK layer ends up as 430MB. Maybe a trimmer eureka will happen one day, but it isn't terribly important.